02.Gil.local + 03.Git.Hosting v2

BrazovskyVladimir opened the pull request.

pluhin commented on .gitlab-ci.yml:
```diff
@@ -0,0 +1,13 @@
+image: ubuntu
```
All tour files should be in your folder, can you move it

pluhin commented on VladimirBrazovsky/03.Git.Hosting/03.Git.hosting.md:
```diff
@@ -0,0 +1,8 @@
+orig_bit_braz	ssh://git@bitbucket.org/brazovsky/brazovsky.git (fetch)
```
Please, use git markup for this addresses

BrazovskyVladimir commented:
I changed

BrazovskyVladimir commented:
I replaced 02.Git.local to README.md in 02.Git.local folder. But I made mistake when created first repository and my "master" branch hid under your "master" after rebase. I renamed my branch, but all commit were lose. I can show screenshot from GitKraken.   (Пишу в спешке, прошу простить за кривой English)
